AsyncDisplayKit/Layout/ASStaticLayoutSpec.h

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import <AsyncDisplayKit/ASLayoutSpec.h>
+#import <AsyncDisplayKit/ASStaticLayoutSpecDimension.h>
+
+@interface ASStaticLayoutSpecChild : NSObject
```

Comment:
Is this the ideal way to create a manual layout spec that allows full control of positioning the elements, but allows the subclass to have no custom implementation of -layout?  I wonder if there is a shorter / simpler name for this.  It probably makes more sense to name ASManualLayoutSpec, or at least clarify comments referring to some use cases.